src/cli.rs

```diff
@@ -8,7 +8,7 @@ use clap::{Parser, Subcommand};
 pub(crate) struct GptcommitCLI {
     /// The action to perform (subcommand).
     #[command(subcommand)]
-    pub action: Action,
+    pub action: Option<Action>,
```

Comment:
Rather than making this optional, add a new Action type which can be the default. I'd recommend `-` for the action to match other CLI tools like `cat - `